pages/common/ssh-add.md

```diff
@@ -0,0 +1,23 @@
+# ssh-add
+
+> Manage loaded RSA or DSA keys in the ssh-agent.
+
+- Add a new key to the ssh-agent:
```

Comment:
Start with the simplest command invocation `ssh-add` and explain what it does. Then move on to adding a given key to the agent.

Reply:
didn't know that ssh-add had also a optionless function. One never stops learning.